easybuild/easyconfigs/p/Pillow/Pillow-3.2.0-foss-2016a-Python-3.5.1.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name = 'Pillow'
+version = '3.2.0'
+versionsuffix = '-Python-%(pyver)s'
```

Comment:
@hajgato why not use the existing `Pillow-3.2.0-intel-2016a-Python-2.7.11-freetype-2.6.3.eb,`?